src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/resource_allocation_manager.h

```diff
@@ -120,6 +120,10 @@ class ResourceAllocationManager {
    */
   virtual void SendOnRCStatusNotification() = 0;
 
+  virtual bool is_rc_enabled() const = 0;
+
+  virtual void set_is_rc_enabled(const bool value) = 0;
```

Comment:
@LitvinenkoIra REbase to @BSolonenko branch or cherry-pick necessary commits. There are functions with proper naming

Reply:
@AByzhynar you mean this functions? https://github.com/smartdevicelink/sdl_core/commit/900055a4b78449f61915980999ad210ba2c4b48e
is_rc_enabled - checks that RC functionality allowed (by user via HMI)
is_remote_control_supported - checks that app registered with AppHMIType::REMOTE_CONTROL

Reply:
@LitvinenkoIra Right. I meant those functions. But as these functions are different - rename properly

Reply:
@abyzhynar @ilitvinenko there is no need to add additional profile value for it. RC is enabled by default by requirements. In case if we won't to disable RC we always are able to send RPC. I don't see use case of modifying this default value

Reply:
@AByzhynar renamed function 94a277ba57f8412d23e023946a2568290ecc86c2